### DISCUSSION TITLE
Relation with multiple select - how show pre-selected values

### DISCUSSION THREAD

**Discussion by info170** [orchidsoftware/platform#2753] [Q&A] [answered]
Please help, I couldn't find the solution.
I got the model News, model Project and many2many table news_project.
And the Layout Relation
`
                Relation::make('news.project_id.')
                    ->fromModel(Project::class, 'code')
                    ->multiple()
                    ->title('Project')
`
It works properly for first time selection and storing data.
But how describe make parameter to show pre-selected multiple values project_id if it stores in many2many table (not in news table) ?

**Comment by macit-emre** [comment, marked as answer]
i think you should use plural form of project_id as 'news.project_ids.' then you should load your model with that project_ids using 'with' in your model public function project_ids(){ return $this->belongsToManyBlaBla() } then this will probably show up. 

A little workaround is, load your ids and their labels by hand, then give Relation::make(blabla)...->value($idsFetchedByHand)

**Reply by info170** [reply]
Thanks a lot.
What I was need to know - "value" method.
And now it works properly in such syntax:
`
>                 Relation::make('news.project_id.')
>                     ->fromModel(Project::class, 'code')
>                     ->value($this->news->projects()->pluck('project_id'))
>                     ->multiple()
>                     ->title('Project'),
`
